lenovo/thinkpad/x1/default.nix

```
@@ -1,6 +1,9 @@
+{ lib, pkgs, ... }:
+
 {
   imports = [
     ../.
     ../../../common/cpu/intel
   ];
+  boot.kernelPackages = pkgs.linuxPackages_latest;
```

Let's make this future proof.

```suggestion
  # This solves lagging noticeable on high-resolution screens.
  boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "5.15")  pkgs.linuxPackages_latest;
```